packages/react/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@axa-fr/react-oidc",
   "version": "6.15.8",
   "private": false,
```

Comment:
@jafin ,

The package.json in vanilla folder have to be adapted too.

I would like to set up back learna but I'am lacking of time : https://github.com/lerna/lerna
To set it up quickly, i copied file from react to vanilla in order to publish vanilla library.

Reply:
@guillaume-chervet thanks I'll investigate lerna and get back with an update.

Reply:
Vanilla lib just need and update package.json that copy and publish the new service worker folder already compiled from react folder.

Reply:
Sorry, I misunderstood the intent of your comments.  I will amend then with the changes you have suggested.  Leaving lerna as a separate PR.

Reply:
@guillaume-chervet I've amended the vanilla package to copy the compile oidcServiceWorker.js from React package.
It's a bit of a muddle since vanilla doesn't clarify that react/service_worker needs to be compiled. I think following up with a Lerna/workspaces to enable package dependency should reduce the complication. I'll continue to look as this as a separate PR improvement.